client/src/components/ViewBlog/ViewBlog.jsx

```diff
@@ -14,10 +18,21 @@ const ViewBlog = (props) => {
                 <h1 className={styles.title}>{title}</h1>
                 <span className={styles.description}> {description || defaultDescription} </span>
                 <div className={styles.blogDetails}>
-                    <UIAvatar name='John Snow' size={50} rounded className={styles.left} />
+                    <UIAvatar 
+                    name={userId.name} 
+                    size={50} 
+                    rounded 
+                    className={styles.left} 
+                    onClick={() => openPathInNewTab(`/profile/${userId.username}`)} />
```

Comment:
Wrapping this in the Link would be better. Because it will increase accessibility.

Reply:
Also, you can use target="_blank"

Reply:
done